test/common/types/utility_test.cc

```diff
@@ -0,0 +1,24 @@
+#include "gtest/gtest.h"
+#include "library/common/data/utility.h"
+#include "library/common/types/c_types.h"
+#include "library/common/types/utility.h"
+
+namespace Envoy {
+namespace Types {
+
+TEST(EnvoyMapConvenientInitializerTest, FromCppToCEmpty) {
```

Comment:
this file probably belong here: https://github.com/envoyproxy/envoy-mobile/tree/main/test/common/data

Reply:
moved to `test/common/bridge` per suggestion above.